ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2

```diff
@@ -159,6 +159,35 @@
 {%         endif %}
 {%     endif %}
 {% endif %}
+{# Gather overlay peering information #}
+{% set switch.evpn_overlay_peers = [] %}
+{# #}
+{# Look for evpn_overlay_controllers / servers where we are client #}
+{% if evpn_overlay_controller_groups is defined and evpn_overlay_controller_groups is not none %}
+{%     for evpn_overlay_controller_group in evpn_overlay_controller_groups %}
+{%         for some_switch in groups[evpn_overlay_controller_group] %}
```

Comment:
Maybe we should use a more specific name to relate `some_switch` ? Like `evpn_overlay_switch` as it is only iteration over `groups[evpn_overlay_controller_group]`